src/plugins/webPWA.web/index.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { disableStyle, enableStyle } from "@api/Styles";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { findStoreLazy } from "@webpack";
+import { RelationshipStore } from "@webpack/common";
+import type { FluxStore } from "@webpack/types";
+
+import style from "./style.css?managed";
```

Comment:
file name mismatches
```suggestion
import style from "./styles.css?managed";
```